icu4c/source/test/fuzzer/fuzzer_driver.cpp

```diff
@@ -10,11 +12,49 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size);
 
 int main(int argc, char* argv[])
 {
-  (void) argc;
-  (void) argv;
-  const char *fuzzer_data = "abc123";
-   
-  LLVMFuzzerTestOneInput((const uint8_t *) fuzzer_data, strlen(fuzzer_data));
+  bool show_warning = true;
+#if UPRV_HAS_FEATURE(address_sanitizer)
+  show_warning = false;
+#endif
+#if UPRV_HAS_FEATURE(memorys_sanitizer)
+  show_warning = false;
+#endif
+  if (argc > 2 && strcmp(argv[2], "-s") == 0) {
```

Comment:
What is "s"? "silent"? We usually use "-q" for "quiet".

You need not use the ICU options library, but it would be good to be consistent.
https://github.com/unicode-org/icu/blob/master/icu4c/source/tools/toolutil/uoptions.h